### PR TITLE
Setting up devise

### PR DESCRIPTION
**Pull request by wazzuper**
- Installed a devise and omniauth gems
- Configured devise
- Generated migration
- Generated devise views with 'devise-bootstrap-views' gem

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
```
Comment (atatarenko) [inline app/models/user.rb]: Let's remove that comment. It doesn't really bring any sense.

File: config/locales/devise.views.it.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
+it:
```
Comment (atatarenko) [inline config/locales/devise.views.it.yml]: Wow, Italian :)
I like Italian but do we really need this locale in our app?

File: db/migrate/20180226045505_devise_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
+      t.timestamps null: false
+    end
+
+    add_index :users, :email,                unique: true
+    add_index :users, :reset_password_token, unique: true
+    add_index :users, :confirmation_token,   unique: true
+    # add_index :users, :unlock_token,         unique: true
```
Comment (atatarenko) [inline db/migrate/20180226045505_devise_create_users.rb:41]: If you commented this line manually, guess, we can remove it.
If it was autogenerated, that's fine, leave as is.

**Review by atatarenko — APPROVED** [review]
LGTM

If there are a frontend works, attach a screenshot or gif displaying changes.

File: config/locales/devise.views.it.yml (hunk 2 above)
Comment (wazzuper) [reply config/locales/devise.views.it.yml]: Oops, missed that file. Deleted.

File: db/migrate/20180226045505_devise_create_users.rb (hunk 3 above)
Comment (wazzuper) [reply db/migrate/20180226045505_devise_create_users.rb:41]: It was autogenerated.

**Comment by wazzuper** [comment]
Here is a screenshot with devise forms after adding a devise-bootstrap-views gem

screenshot from 2018-02-27 13-34-34

**Pull request merged**